src/components/index.ts

```diff
@@ -5,5 +5,5 @@ export { Image } from './Image';
 export { RedBox } from './RedBox';
 export { View } from './View';
 export { Text } from './Text';
-import * as Svg from './Svg';
+import Svg from './Svg';
```

Comment:
Am happy to revert this if requested. Am doing this to fix the `basic-svg` example to unblock.